samples/System.IO.Pipelines.Samples/SampleBase/RawHttpClientSampleBase.cs

```diff
@@ -14,7 +15,7 @@ public abstract class RawHttpClientSampleBase : ISample
     {
         public async Task Run()
         {
-            var consoleOutput = GetPipeFactory().CreateWriter(Console.OpenStandardOutput());
+            var consoleOutput = StreamPipeConnection.CreateWriter(new PipeOptions(GetBufferPool()), Console.OpenStandardOutput());
```

Comment:
Messy

Reply:
Could we default to some pool instead of having to always pass it in?

Reply:
It has to be disposed, making options `IDisposable` would be a bit strange.